packages/accounts-password/password_server.js

```diff
@@ -538,44 +538,89 @@ Accounts.sendResetPasswordEmail = function (userId, email) {
   if (!email || !_.contains(_.pluck(user.emails || [], 'address'), email))
     throw new Error("No such email for user.");
 
+  var tokenReason;
```

Comment:
I fully understand your idea of matching surrounding code, but based on that logic you should have used brace-less `if`/`else` conditionals below (for `tokenReason`) (I'm not in favor of that, just saying)

Shouldn't we incrementally move toward more commonly preferred syntax/styles whenever possible?  I don't know how far we can stretch the "surrounding"-style logic – where does it stop?  The function?  The function's function?  The parent class?  The file?